src/metabase/api/dataset.clj

```diff
@@ -176,22 +177,18 @@
       (qp.pivot/run-pivot-query (assoc query :async? true) info context))))
 
 
-(def ^:private Parameter
```

Comment:
@dpsutton any reason why you used this single with a single field instead of the full schema we have in `metabase.util.malli.schema`?